src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -854,6 +862,190 @@ protected function send_email( $type, $core_update, $result = null ) {
+			$this->send_plugin_theme_email( 'success', $successful_updates, $failed_updates );
+		} elseif ( empty( $successful_updates ) ) {
+			$this->send_plugin_theme_email( 'fail', $successful_updates, $failed_updates );
+		} else {
+			$this->send_plugin_theme_email( 'mixed', $successful_updates, $failed_updates );
+		}
+	}
+
+	/**
+	 * Sends an email upon the completion or failure of a plugin or theme background update.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param string $type               The type of email to send. Can be one of 'success', 'failure', 'mixed'.
+	 * @param array  $successful_updates A list of updates that succeeded.
+	 * @param array  $failed_updates     A list of updates that failed.
+	 */
+	protected function send_plugin_theme_email( $type, $successful_updates, $failed_updates ) {
+		// No updates were attempted.
+		if ( empty( $successful_updates ) && empty( $failed_updates ) ) {
+			return;
+		}
+		$body = array();
+
+		switch ( $type ) {
+			case 'success':
+				/* translators: %s: Site title. */
+				$subject = __( '[%s] Some plugins or themes were automatically updated' );
+				break;
+			case 'fail':
+				/* translators: %s: Site title. */
+				$subject = __( '[%s] Some plugins or themes have failed to update' );
+				$body[]  = sprintf(
+					/* translators: %s: Home URL. */
+					__( 'Howdy! Failures occurred when attempting to update plugins/themes on your site at %s.' ),
+					home_url()
+				);
+				$body[] = "\n";
+				$body[] = __( 'Please check out your site now. It’s possible that everything is working. If it says you need to update, you should do so.' );
+				break;
+			case 'mixed':
+				/* translators: %s: Site title. */
+				$subject = __( '[%s] Some plugins or themes were automatically updated' );
+				$body[] = sprintf(
+					/* translators: %s: Home URL. */
+					__( 'Howdy! Failures occurred when attempting to update plugins/themes on your site at %s.' ),
+					home_url()
+				);
+				$body[] = "\n";
+				$body[] = __( 'Please check out your site now. It’s possible that everything is working. If it says you need to update, you should do so.' );
+				$body[] = "\n";
+				break;
+		}
+
+		// Get failed plugin updates.
+		if ( in_array( $type, array( 'fail', 'mixed' ), true ) && ! empty( $failed_updates['plugin'] ) ) {
+			$body[] = __( 'The following plugins failed to update:' );
+			// List failed updates.
+			foreach ( $failed_updates['plugin'] as $item ) {
+				/* translators: %s: Name of the related plugin. */
+				$body[] = ' ' . sprintf( __( '- %s' ), $item->name );
+			}
+			$body[] = "\n";
+		}
+		// Get failed theme updates.
+		if ( in_array( $type, array( 'fail', 'mixed' ), true ) && ! empty( $failed_updates['theme'] ) ) {
+			$body[] = __( 'The following themes failed to update:' );
+			// List failed updates.
+			foreach ( $failed_updates['theme'] as $item ) {
+				/* translators: %s: Name of the related plugin. */
+				$body[] = ' ' . sprintf( __( '- %s' ), $item->name );
```

Comment:
```suggestion
				$body[] = ' ' . sprintf( '- %s', $item->name );
```

There's nothing to translate.

Reply:
Thanks, this was fixed in [PR 138](https://github.com/WordPress/wp-autoupdates/pull/138). We'll update the core patch accordingly.